android/src/main/assets/google-services.json

```diff
@@ -0,0 +1,55 @@
+{
+  "project_info": {
+    "project_number": "606957953509",
+    "firebase_url": "https://farmers-tmp.firebaseio.com",
```

Comment:
farmers-tmp? :)
How about you open a new Firestore project for this demo app (startupos-android-example)?

Or maybe we shouldn't include a google-services.json. WDYT?

Reply:
@oferb 
>Or maybe we shouldn't include a google-services.json. WDYT?
it got into PR by mistake 🙂

Reply:
How about we add it to .gitignore then?